code/game/objects/structures/morgue.dm

```diff
@@ -354,12 +336,58 @@ GLOBAL_LIST_EMPTY(crematoriums)
 	add_fingerprint(user)
 	update_icon(UPDATE_OVERLAYS)
 
-/obj/structure/crematorium/attackby(P as obj, mob/user as mob, params)
+/obj/structure/crematorium/obj_break(damage_flag)
```

Comment:
Can you still open and close it when broken? If not, you should probably spit out the contents to stop it being used as some weird unescapeable cell

Reply:
It opens and closes just fine while broken, it just can't be turned on.